test/integration/narrow/routes/reset-payment-request.test.js

```diff
@@ -0,0 +1,69 @@
+jest.mock('../../../../app/plugins/crumb')
+jest.mock('../../../../app/api')
+const { post } = require('../../../../app/api')
+jest.mock('../../../../app/auth')
+const createServer = require('../../../../app/server')
+const { schemeAdmin } = require('../../../../app/auth/permissions')
+const Boom = require('@hapi/boom')
+
+const url = '/payment-request/reset'
+const validInvoiceNumber = 'S1234567S123456V001'
+let server
+let auth
+
+describe('Reset payment request', () => {
+  beforeEach(async () => {
+    auth = { strategy: 'session-auth', credentials: { scope: [schemeAdmin] } }
+    jest.clearAllMocks()
+    server = await createServer()
+  })
+
+  afterEach(async () => {
+    await server.stop()
+  })
+
+  test('returns 403 and if no permissions in scope', async () => {
+    auth.credentials.scope = []
+    const res = await server.inject({ method: 'GET', url, auth })
+    expect(res.statusCode).toBe(403)
+  })
+
+  test('returns 302 if not authorised', async () => {
+    const res = await server.inject({ method: 'GET', url })
+    expect(res.statusCode).toBe(302)
+    expect(res.headers.location).toEqual('/login')
+  })
+
+  test('returns 200 page if authorised', async () => {
```

Comment:
`returns 200 if authorised` for consistency -- plus the test isn't checking if its a page returned